### PR TITLE
[Core] Access PolynomialDegree() of QuadraturePointGeometry

### PR DESCRIPTION
**Pull request by tteschemacher**
Yoo yoo yoo yoo yoo

This allows the access to the polynomial degree through the quadrature point.

AND IT HAS A UNIT TEST 💯

### CODE REVIEW

**Review by dagmawibekel — DISMISSED** [review]

File: kratos/geometries/quadrature_point_geometry.h [hunk 1]
```diff
@@ -280,6 +280,20 @@ class QuadraturePointGeometry
...
+    ///@}
+    ///@name Mathematical Informations
+    ///@{
+
+    /// Returns the polynomial degree of the parent geometry
+    SizeType PolynomialDegree(IndexType LocalDirectionIndex) const override
+    {
+        KRATOS_DEBUG_ERROR_IF(mpGeometryParent == nullptr)
+            << "Trying to call PolynomialDegree(LocalDirectionIndex) from quadrature point. "
+            << "Pointer to parent is not assigned." << std::endl;
```
Comment (philbucher) [inline kratos/geometries/quadrature_point_geometry.h]:
```suggestion
        KRATOS_DEBUG_ERROR_IF_NOT(mpGeometryParent)
            << "Trying to call PolynomialDegree(LocalDirectionIndex) from quadrature point. "
            << "Pointer to parent is not assigned." << std::endl;
```

**Review by philbucher — DISMISSED** [review]

File: kratos/geometries/quadrature_point_geometry.h (hunk 1 above)
Comment (tteschemacher) [reply kratos/geometries/quadrature_point_geometry.h]: are you sure that works?

Comment (philbucher) [reply kratos/geometries/quadrature_point_geometry.h]: [yes](https://stackoverflow.com/a/17772117)

Comment (tteschemacher) [reply kratos/geometries/quadrature_point_geometry.h]: a yes with a link ❤️

**Review by philbucher — APPROVED** [review]

**Pull request merged**